images/releaser/Dockerfile

```
@@ -8,6 +8,7 @@ FROM alpine:3.19
 COPY --from=builder /go/bin/crane /usr/local/bin/crane
 COPY --from=builder /go/bin/cosign /usr/local/bin/cosign
 COPY release.sh /usr/local/bin/release
+SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 
```

I think it has to be one line below
```suggestion

SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
```